lib/msf/ui/console/stylers/rank_styler.rb

```diff
@@ -0,0 +1,17 @@
+
+class RankStyler
+  include Msf
```

Comment:
what's this included for?

Reply:
Included to allow the RankStyler to access the Constants in `lib/msf/core/constants.rb`. Is the above a smell?

Reply:
Just reference the constants with `Msf::` prefixed.

Reply:
yup @wvu-r7 beat me to it